### DISCUSSION TITLE
No v1.18.1 in Home Assistant OS 5.12?

### DISCUSSION THREAD

**Discussion by GD-Dal** [Koenkk/zigbee2mqtt#6548] [General]
Is there a reason, or should I be able to see the v.1.18.1 i Home Assistant OS 5.12?
I cannot see any updates available, and I'm running v1.17.4

Is it not pushed yet, or is it something wrong with my installation?

Thanks.

**Comment by m0wlheld** [comment]
Better check the [issues](https://github.com/zigbee2mqtt/hassio-zigbee2mqtt/issues) at the add-on site.

**Reply by GD-Dal** [reply]
It seems that the URL for the repository has changed?
As soon as I typed in https://github.com/zigbee2mqtt/hassio-zigbee2mqtt, the new version showed up.
Unfortunately, that seemed to make an orphan out of the current v1.17 install.
In other words, there are no upgrade option, just an install option if I want to upgrade. 
What will happen to the connected nodes then?
Unfortunatly, I don't remember the old URL so I can revert this. 
Anyone?

**Reply by GD-Dal** [reply]
Can anyone confirm that Zigbee2MQTT is split into 2 versions / repositories now?
Anyway, I'm not able to make v1.18.1 to start.
I'm getting errors like these:
`[12:23:45] INFO: Handing over control to Zigbee2mqtt Core ...
> zigbee2mqtt@1.18.1 start /app
> node index.js
Zigbee2MQTT:error 2021-03-05 12:24:05: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2021-03-05 12:24:05: Failed to start zigbee
Zigbee2MQTT:error 2021-03-05 12:24:05: Exiting...
Zigbee2MQTT:error 2021-03-05 12:24:05: Error: Failed to connect to the adapter (Error: SRSP - SYS - ping after 6000ms)
    at ZStackAdapter.<anonymous> (/app/node_modules/zigbee-herdsman/dist/adapter/z-stack/adapter/zStackAdapter.js:94:31)
    at Generator.throw (<anonymous>)
    at rejected (/app/node_modules/zigbee-herdsman/dist/adapter/z-stack/adapter/zStackAdapter.js:25:65)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! zigbee2mqtt@1.18.1 start: `node index.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the zigbee2mqtt@1.18.1 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-03-05T11_24_05_858Z-debug.log`
